lib/cocina/models/collection_attributes.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Cocina
+  module Models
+    # Attributes, types and Structs common to Vocab.collection objects (plain and request)
+    # See http://sul-dlss.github.io/cocina-models/maps/Collection.json
+    module CollectionAttributes
+      include Checkable
+
+      TYPES = [
+        Vocab.collection,
+        Vocab.curated_collection,
+        Vocab.exhibit,
+        Vocab.series,
+        Vocab.user_collection
+      ].freeze
+
+      # Subschema for access concerns
+      class Access < Struct
+        attribute :access, Types::String.default('dark')
+                                        .enum('world', 'stanford', 'location-based', 'citation-only', 'dark')
+      end
+
+      # Subschema for administrative concerns
+      class Administrative < Struct
+        attribute :releaseTags, Types::Strict::Array.of(ReleaseTag).default([].freeze)
+        # Allowing hasAdminPolicy to be omittable for now (until rolled out to consumers),
+        # but I think it's actually required for every DRO
+        attribute :hasAdminPolicy, Types::Coercible::String.optional.default(nil)
+      end
+
+      # Identification sub-schema for the Collection
+      class Identification < Struct
+        attribute :catalogLinks, Types::Strict::Array.of(CatalogLink).meta(omittable: true)
+      end
+
+      class Structural < Struct
+      end
```

Comment:
Please remove these, they aren't part of the shared attributes.

Reply:
where do you think they should be declared, to avoid duplication?

Reply:
They weren’t duplicated before this PR. So the can be left where they are

Reply:
ok, thanks for the explanation